client/src/controllers/index.ts

```diff
@@ -22,6 +22,7 @@ import { TagController } from './TagController';
 import { TeleportController } from './TeleportController';
 import { TooltipController } from './TooltipController';
 import { UpgradeController } from './UpgradeController';
+import { TabsController } from './TabsController';
```

Comment:
Please order the imports and registrations alphabetical - as per the comments in the file.